backend/battles/views.py

```diff
@@ -36,3 +38,14 @@ class BattleDetail(DetailView):
 
 class BattleList(ListView):
     model = Battle
+
+
+class SettledBattlesList(ListView):
+    model = Battle
+
+    def get_queryset(self):
+        return Battle.objects.filter(
```

Comment:
I suggest you set this filter in the model manager so whenever you want to filter for settled battles you can just do `Battle.objects.settled()`

Reply:
Check this: https://docs.djangoproject.com/en/3.1/topics/db/managers/#creating-a-manager-with-queryset-methods